autogpt/agent/agent.py

```diff
@@ -14,6 +14,48 @@
 from autogpt.workspace import Workspace
 
 
+def should_prompt_user(continuous_mode, next_action_count):
```

Comment:
Make this a staticmethod on the agent.  It belongs in the agent's namespace. 

```

def should_prompt_user(self):

Reply:
All three of these functions should be made staticmethods.  They should be proper methods, but we can address that after the functionality is in.